numpy/core/src/private/ufunc_override.c

```diff
@@ -8,148 +8,172 @@
 
 #include "ufunc_override.h"
 
-static void
+
+static int
 normalize___call___args(PyUFuncObject *ufunc, PyObject *args,
-                    PyObject **normal_args, PyObject **normal_kwds,
-                    int nin)
+                        PyObject **normal_args, PyObject **normal_kwds,
+                        int nin, int nout)
```

Comment:
If we're passing non-ufuncs in, then we should change the signature to `PyObject*` to make it clear that no properties can be depended on.